common-arcana.lic

```diff
@@ -537,88 +537,6 @@ module DRCA
     end
   end
 
-  def update_astral_data(data)
-    if data['moon']
-      data = set_moon_data(data)
-    elsif data['stats']
-      data = set_planet_data(data)
-    end
-    data
-  end
-
-  def find_visible_planets(planets)
```

Comment:
In this repo only there are some test scripts. `test/test_common_arcana.rb` will need to swap from using DRCA to DRCMM please, thanks!